scripts/README.md

```diff
@@ -34,6 +34,16 @@ understand how the BringAuto software and hardware platform works.
 - is a submodule so it needs to be initialized by:
 - - `git submodule update --init --recursive`
 
+### create_docker_compose_for_testing
+
+- creates a new docker-compose.yml with docker images replaced by desired images built from local repositories
+- run the script by `python3 create_docker_compose_for_testing.py`
+- the config file `docker_compose_for_testing.json` needs to be adjusted
```

Comment:
Are you sure about the sequence of ops? Should I not update the config and then run the script?

Reply:
Adjusted the readme for scripts, so that the script execution examples are last in order